hud-data/weirdcorehud.json

```
@@ -5,7 +5,7 @@
   "releaseDate": "2024-04-06",
   "resources": [
     "sfondo-hud",
-    "https://youtu.be/hlAn1QJPX7o",
+    "https:///youtu.be/QUye8KZD2hU",
```

```suggestion
    "https://youtu.be/QUye8KZD2hU",
```
mcoms accidentally added an extra /. It does still work like this, but just to be consistent